src/popup/locales/en.json

```
@@ -79,43 +79,43 @@
       "msg": "If a URL is marked as verified, it means the owner of this URL is already a fellow Superhero — and that is awesome! An AE address has been found on the page and tips can be claimed easily."
     },
     "not-verified": {
-      "title": "What is \"Not Verified\" URL?",
+      "title": "What does it mean when a URL is \"Not Verified\"?",
       "msg": "This means that the owner of the URL has not created a Superhero wallet yet. They can claim their tips as soon as they make one and add their wallet address to their website. "
     },
-    "blacklisted": {
-      "title": "What is \"Blacklisted\" URL?",
+    "inaccessible": {
```

```suggestion
    "blacklisted": {
```

This should not be changed as this is used the key.